testcases/PowerNVDump.py

```diff
@@ -305,6 +309,76 @@ def runTest(self):
         return boot_type
 
 
+class SBECrash_MPIPL(PowerNVDump):
+
+    def runTest(self):
+
+        if "OpenBMC" in self.bmc_type:
+            cmd = "mboxctl --reset"
+        elif "SMC" in self.bmc_type:
+            raise self.skipTest(
+                "SBE initiated MPIPL is not supported on SMC yet")
+        #run mboxctl reset on BMC
+        self.cv_BMC.run_command(cmd)
+        self.c.run_command("uname -a")
+        self.c.run_command("cat /etc/os-release")
```

Comment:
Why you want to run above two commands?

Reply:
Nothing related to MPIPL. But justed wanted that information, so in case of failure we will get kernel version and os information in logs.